### PR TITLE
IA-1001: OrgUnit detail cypress testing

### PR DESCRIPTION
**Pull request by beygorghor**
- writing cutom cyoress commands for select, multiselect, text input, array field
- adding custom id's for testing
- testing complete change and save of orgUnit infos

https://user-images.githubusercontent.com/12494624/152552469-2d44eb89-5f49-4a02-9384-05f22b283e9f.mov

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewPicker.js [hunk 1]
```diff
@@ -109,14 +111,19 @@ const OrgUnitTreeviewPicker = ({
...
             <Paper variant="outlined" elevation={0} className={className}>
                 {makeTruncatedTrees(selectedItems)}
                 {clearable && resetSelection && selectedItems.size > 0 && (
-                    <Box className={classes.clearButton}>
+                    <Box
+                        className={classnames(
+                            classes.clearButton,
+                            'clear-tree',
+                        )}
+                    >
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewPicker.js:126]: This needs to be updated in the bluesquare-components version as well (or will be when we switch)

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewPicker.js:126]: same with the class on `InputLabel` I think

File: hat/assets/js/cypress/support/commands.js [hunk 2]
```diff
@@ -81,3 +86,96 @@ Cypress.Commands.add(
...
+/**
+ * @param {number} id - Base id used to select DOM element
+ * @param {array} selectedOptions - list of options selected ids
+ */
+Cypress.Commands.add('fillMultiSelect', (id, selectedOptions = []) => {
+    cy.get(id).as('multiSelect');
+    cy.get('@multiSelect').click();
+    cy.get('@multiSelect')
+        .parent()
+        .find('.MuiAutocomplete-clearIndicator')
```
Comment (quang-le) [inline hat/assets/js/cypress/support/commands.js:138]: I'm not keen on adding classes for the sake of it, but using MUI classes for our cypress commands will break the test if we decide to change our implementation of multiselect.

**Review by quang-le — COMMENTED** [review]
Nice job with the commands. A bit of nitpicking and splitting hair from me as usual 😉

File: hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewPicker.js (hunk 1 above)
Comment (beygorghor) [reply hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewPicker.js:126]: This has been merged already [here](https://github.com/BLSQ/bluesquare-components/pull/38)

File: hat/assets/js/cypress/support/commands.js (hunk 2 above)
Comment (beygorghor) [reply hat/assets/js/cypress/support/commands.js:138]: Yes that's also why i'm doing it only in the commands normally...

**Pull request merged**